extension/content.js

```diff
@@ -113,6 +113,24 @@ function addTrendingMenuItem() {
 	`);
 }
 
+function addYoursMenuItem() {
+	const [, page] = location.pathname.split('/');
+	const username = getUsername();
+	const $menu = $('.subnav-links');
+
+	if ($(`.subnav-item[href="/${page}?q=is%3Aopen+is%3Aissue+user%3A${username}"]`).length > 0) {
```

Comment:
I'd rather use a dedicated class for this (by adding a `refined-github-yours-tab` class to the button and checking for that)